Tribler/Main/tribler_main.py

```diff
@@ -943,7 +943,6 @@ def OnExit(self):
             self._logger.info("ONEXIT Session is shutdown")
 
         self.utility.session.notifier.notify(NTFY_CLOSE_TICK, NTFY_INSERT, None, 'Deleting instances')
-        wx.Yield()
```

Comment:
It hang on it, @devos50 seen it happen earlier iirc. WX is not letting go apparently.